Gopkg.toml

```diff
@@ -43,24 +43,24 @@
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
   name = "github.com/giantswarm/operatorkit"
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.1"
```

Comment:
Strangely this one was older compared to other k8s libraries. If anyone knows/remembers what was rationale for that, please share, I'm curious whether same rationale still holds.

Reply:
Not sure. It could be because this library was created recently.

In general we want everything to be on the same version.